scripts/e2e-wdio-saucelabs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+export CI=true
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+setup_service java 1.8.222
+setup_service google-chrome-stable 89.0.4389.72-1
+
+export RUN_SAUCE_TESTS=true
+export SAUCE_USERNAME=OktaSignInWidget
+get_vault_secret_key devex/sauce-labs accessKey SAUCE_ACCESS_KEY
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/e2e-wdio-saucelabs"
+
+# This file contains all the env vars we need for e2e tests
+aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/signinwidget/export-test-credentials.sh $OKTA_HOME/$REPO/scripts/export-test-credentials.sh
+source $OKTA_HOME/$REPO/scripts/export-test-credentials.sh
+
+# We use the below OIE enabled org and clients for OIE tests
+export WIDGET_TEST_SERVER=https://oie-widget-tests.sigmanetcorp.us
```

Comment:
We should use the new production org instead. This org is not behaving as expected. 
https://github.com/okta/okta-signin-widget/blob/master/scripts/e2e-wdio.sh#L19